vm-virtio/src/balloon.rs

```diff
@@ -304,14 +292,6 @@ impl BalloonEpollHandler {
         )
         .map_err(DeviceError::EpollCtl)?;
 
-        epoll::ctl(
-            epoll_file.as_raw_fd(),
-            epoll::ControlOptions::EPOLL_CTL_ADD,
-            self.update_memory_evt.as_raw_fd(),
-            epoll::Event::new(epoll::Events::EPOLLIN, u64::from(UPDATE_MEMORY_EVENT)),
-        )
-        .map_err(DeviceError::EpollCtl)?;
-
```

Comment:
@teawater This commit simply reverts the previous patch, but of course both commits must be removed before we can merge the PR. I just wanted to keep it this way so that you can take care of it.